### ISSUE TITLE
Add the ability to add custom Ethereum ERC-20 tokens

### ISSUE THREAD

**Issue by MartinacPlus** [LedgerHQ/ledger-live-desktop#3276]
- I have recently sent "BTC Network Demand Set (BYTE)" tokens to my Ethereum wallet address, but these tokens are not visible in Ledger Live app. Reason is that the BYTE token is not supported yet.

- I think the option to manually add any ER-20 tokens should be added. To add any custom token would be then simple and there would be no need for it to be explicitly supported. I have manually added the BYTE token to Metamask using just its [contract address](https://etherscan.io/address/0xac8ea871e2d5f4be618905f36f73c760f8cfdc8e). No problem with that, but it would be neat to have Ledger Live app as a one-stop shop for all of one's crypto needs.

#### Part of the application

- Ethereum account section.

**Comment by gre** [comment]
It's not that simple because you need Ledger's signature to properly display an ERC20 on the device. The feature is still considered but it would be a message to sign on the device and you would not see the token and the amount without a signature. This is the main reason we don't have "all possible" tokens.

**Comment by MartinacPlus** [comment]
Sorry, I'm not sure I understood. Now I use the MetaMask's Firefox plugin to which I authenticate with my Ledger Nano devices. I'm able to sign transactions, see tokens and Ethereum balances. Why the same thing cannot be done on Ledger Live? Or is the way it is implemented on MetaMask unsafe?

**Comment by devli13** [comment]
@MartinacPlus  Agreed with this comparison brought up regarding metamask. I have been told by Ledger staff that this functionality is being worked on for the enterprise solution but would be great to have for consumers to easily add tokens by smart contract.

@gre , I think many would be okay with all the message details on the ledger device given interactions with smart contracts already offer limited transparency of the metadata on the ledger nano device.
